### DISCUSSION TITLE
what is basic diff between react and vue

### DISCUSSION THREAD

**Discussion by RanjithSrikanth** [community/community#134782] [Programming Help]
### Body

what is basic diff between react and vue

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Vishwaa-Arumugam** [comment]
vue is easier than react which will basically follow MVC acrh like ruby on rails 
on the other hand react carry on client and server arch

**Comment by cyberloop001** [comment]
I think Vue is a library that combines the advances of both of them react and angular.

**Comment by ebndev** [comment]
Hi @RanjithSrikanth & @Vishwaa-Arumugam, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
